index.html

```diff
@@ -103,9 +103,42 @@ <h3 class="title-post">Multi-Post Stories</h3>
                                 <li class="tag-4">Ruby</li>
                             </ul>
                         </div>
-                        <button class="button-p-main" type="button">See Project</button>
+                        <button data-modal-target="#modal" class="button-p-main" type="button">See Project</button>
                     </div>
-
+                    <div class="modal" id="modal">
+                        <div class="modal-header">
+                          <div class="title">Keeping track of hundreds  of components website</div>
+                          <button data-close-button class="close-button">&times;</button>
+                        </div>
+                        <div class="modal-body">
+                            <div class="language-tags-model language-tags">
+                                <ul class="tags-model">
+                                    <li class="model-tag tag-1">css</li>
+                                    <li class="model-tag tag-2">html</li>
+                                    <li class="model-tag-b tag-3">bootstrap</li>
+                                    <li class="model-tag tag-4">Ruby</li>
+                                </ul>
+                            </div>
+                            <div class="model-main-project">
+                                <div class="model-Img-Placeholder">
+                                </div>
+                                <div class="model-project-left">
+                                    <div>
+                                        <p class="model-supporting-text">A daily selection of privately personalized reads; no accounts or
+                                            sign-ups
+                                            required. has been the industry's
+                                            standard dummy text ever since the 1500s, when an unknown printer took a standard dummy
+                                            text.</p>
+                                    </div>
+                                    <div class="model-btn">
+                                        <button class="model-button-p-main" type="button">See Project <img src="./Icons/Icon-see-live.png" alt=""></button>
+                                        <button class="model-button-p-main" type="button">See Source <img src="./Icons/github.png" alt=""></button>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <div id="overlay"></div>
                 </div>
                 <div class="card1">
```

Comment:
As the previous Code reviewer specified this card section is supposed to be generated dynamically using Java script DOM manipulation use an array as the previous code reviewer specified. Dont generate the cards usimg HTML.

Please follow this example:
const projects = [{
    name: "Title of Project 1",
    description: "Description of project 1",
    image: "link_to_image.jpg",
    technologies: ["HTML", "JavaScript", "Ruby"],
    link: "link_to_live_version.com",
}]

Reply:
- [ ] As requested by the previous reviewer, kindly ensure that all previous HTML is cleared in the work section, not only for the modal but also the cards. You can then fetch the data for each project from the array on page load in order to have dynamic data fetched for each project. You can use a loop to loop through your array and that will help you dynamically generate the cards.